### PR TITLE
Implement `aggregate` command stub

### LINKED ISSUE #969: Implement `aggregate` command stub
https://www.mongodb.com/docs/manual/reference/command/aggregate/

We should implement a stub that not only returns NotImplemented error but also logs and increments metrics for individual [pipeline stages](https://www.mongodb.com/docs/manual/reference/operator/aggregation-pipeline/). That should help us prioritize future work.

### PR DESCRIPTION
**Pull request by AlekSi**
Closes #969.

## Readiness checklist

* [ ] I added tests for new functionality or bugfixes.
* [x] I ran `task all`, and it passed.
* [x] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [x] I checked comments rendering with `task godocs`.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)),
      Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

**Review by w84thesun — APPROVED** [review]

File: internal/handlers/common/msg_aggregate.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	document, err := msg.Document()
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+
+	pipeline, err := GetRequiredParam[*types.Array](document, "pipeline")
+	if err != nil {
+		return nil, err
+	}
+
```
Comment (noisersup) [inline internal/handlers/common/msg_aggregate.go:40]: When are we using `lazyerrors` and when standard errors?

**Review by noisersup — APPROVED** [review]
LGTM, I have only one question.

File: internal/handlers/common/msg_aggregate.go (hunk 1 above)
Comment (AlekSi) [reply internal/handlers/common/msg_aggregate.go:40]: lazyerrors are more for untyped developer-visible errors. GetRequiredParam returns a good typed error.

**Review by rumyantseva — APPROVED** [review]
LGTM.

I see TODOs are not marked with issue numbers, but I guess it's because you are going to implement them right away.

**Pull request merged**